llvm/test/Transforms/InstCombine/powi.ll

```diff
@@ -258,3 +259,23 @@ define double @different_types_powi(double %x, i32 %y, i64 %z) {
   %mul = fmul reassoc double %p2, %p1
   ret double %mul
 }
+
+define nofpclass(nan inf) double @fdiv_pow_powi(double %x) {
```

Comment:
What is the purpose of the nofpclass here?

Reply:
deleted, thanks